content/en/providers/check-in/sp/_index.md

```diff
@@ -1090,6 +1111,7 @@ The following information about the authenticated user can be provided by EGI
 Check-in in order to control user access to resources:
 
 1. VO/group membership and role information about the authenticated user
+1. GOCDB roles
 1. Level of Assurance (LoA)
```

Comment:
1. VO/group membership and role information
1. Capabilities
1. Identity assurance
1. GOCDB roles